app/forms/csv_upload_form.rb

```diff
@@ -18,7 +22,7 @@ def save(file)
 
     @keywords = ActiveRecord::Base.transaction do
       keywords.map do |keyword|
-        Keyword.create(content: keyword)
+        user.keywords.create(content: keyword)
```

Comment:
While that is possible, the convention is to use the model directly:

```suggestion
        Keyword.create(content: keyword, user: user) 
```

The main reason is that it is will be more consistent in the whole codebase. Usually in the controller, the following code is used:

```
def some_action
   SomeModel.create(some_params)
end

private

def some_params
   params.permit(:param_1, :param_2).merge(user_id: current_user)
end
```

See https://github.com/nimblehq/mia-web/blob/752f45a860b25bdc456c1bcc82b6b5a6ca18e334/engines/mia_payment/app/controllers/mia_payment/bank_accounts_controller.rb